src-electron/generator/matter/app/zap-templates/templates/app/helper.js

```diff
@@ -352,18 +358,51 @@ function chip_endpoint_cluster_list() {
       eventList = `ZAP_GENERATED_EVENTS_INDEX( ${c.eventIndex} )`;
     }
 
+    let individualItems = [];
+    let orderTokens = order.split(',').map((x) => (x ? x.trim() : ''));
+    orderTokens.forEach((tok) => {
+      switch (tok) {
+        case 'clusterId':
+          individualItems.push(`.clusterId = ${c.clusterId}`);
+          break;
+        case 'attributes':
+          individualItems.push(
+            `.attributes = ZAP_ATTRIBUTE_INDEX(${c.attributeIndex})`
```

Comment:
The whitespace here is different from the original code....  might be OK if the consumer runs something like clang-format anyway, or might not be great for readability if not.    @tecimovic

Reply:
This turns out to not be OK on the Matter side; it's causing undesired changes in header outputs.

Reply:
Created https://github.com/project-chip/zap/pull/955